etc/postfix/main.cf

```diff
@@ -59,3 +59,8 @@ sender_canonical_classes = envelope_sender, header_sender
 sender_canonical_maps = regexp:/etc/postfix/sender_canonical
 smtp_header_checks = pcre:/etc/postfix/second_header_checks
 smtpd_data_restrictions = check_sender_access pcre:/etc/postfix/sender_access
+
+## Receive TLS mail (587)
```

Comment:
I think that port `587` is usually used when acting as relay host and doesn't necessarily have to do with tls configuration for the mail server.